exotica_examples/scripts/example_dynamic_time_indexed

```diff
@@ -37,34 +39,45 @@ print('Running with config {0}'.format(configs[problem_idx]))
 solver = exo.Setup.load_solver(configs[problem_idx])
 problem = solver.get_problem()
 solution = solver.solve()
+if solution.shape[0] == 0:
```

Comment:
Maybe write an error/reason why it exits? ["Could not find a solution/did not converge"]

Reply:
Added a print below.